src/main/java/com/proofpoint/discovery/ReplicatedDynamicStore.java

```diff
@@ -44,8 +45,8 @@
     @Inject
     public ReplicatedDynamicStore(@ForDynamicStore DistributedStore store, DiscoveryConfig config)
     {
-        this.store = checkNotNull(store, "store is null");
-        this.maxAge = checkNotNull(config, "config is null").getMaxAge();
+        this.store = requireNonNull(store, "store is null");
+        this.maxAge = Objects.requireNonNull(config, "config is null").getMaxAge();
```

Comment:
Why is this one not statically imported like the others?

Reply:
Somehow missed it.